commit 3a6fad010098c80b3b01928c2473320bef791329 (refs/changes/71/1371/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-01-30T21:58:13+00:00 (2 years, 8 months ago)
    
    joyent/mdb_v8#84 release v1.1.4
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index ce71ca7..15b5496 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -12,6 +12,10 @@
 
 ## Unreleased changes
 
+None.
+
+## v1.1.4 (2017-01-30)
+
 * #77 Node v6.6.0 issues
 * #72: tests cannot run on non-release versions of node
 
diff --git a/README.md b/README.md
index 796b6a5..aabf8b5 100644
--- a/README.md
+++ b/README.md
@@ -44,17 +44,17 @@ https://us-east.manta.joyent.com/Joyent_Dev/public/mdb_v8.  If you have the
 can list the latest binaries with:
 
     $ mfind -t o $(mget -q /Joyent_Dev/public/mdb_v8/latest)
-    /Joyent_Dev/public/mdb_v8/v1.1.3/mdb_v8_amd64.so
-    /Joyent_Dev/public/mdb_v8/v1.1.3/mdb_v8_ia32.so
+    /Joyent_Dev/public/mdb_v8/v1.1.4/mdb_v8_amd64.so
+    /Joyent_Dev/public/mdb_v8/v1.1.4/mdb_v8_ia32.so
 
 You can fetch a specific binary like this (in this case, the 32-bit version
-1.1.3 binary):
+1.1.4 binary):
 
-    $ mget -O /Joyent_Dev/public/mdb_v8/v1.1.3/mdb_v8_ia32.so
+    $ mget -O /Joyent_Dev/public/mdb_v8/v1.1.4/mdb_v8_ia32.so
 
 or using curl:
 
-    $ curl -O https://us-east.manta.joyent.com/Joyent_Dev/public/mdb_v8/v1.1.3/mdb_v8_ia32.so
+    $ curl -O https://us-east.manta.joyent.com/Joyent_Dev/public/mdb_v8/v1.1.4/mdb_v8_ia32.so
 
 This one-liner will get you the latest 32-bit binary:
 
